### ISSUE TITLE
Web Build (jimp.min.js) empty

### ISSUE THREAD

**Issue by ffflorian** [jimp-dev/jimp#936]
# Expected Behavior

Code in `jimp.min.js`.

# Current Behavior

The file contains only the preamble and `undefined`.

# Failure Information (for bugs)

See https://unpkg.com/jimp@0.16.1/browser/lib/jimp.min.js.

The latest version with code in `jimp.min.js` was 0.12.0, see https://unpkg.com/jimp@0.12.0/browser/lib/jimp.min.js. 

## Steps to Reproduce

Open the file.

## Context

- Jimp Version: 0.16.1
- Operating System: Linux
- Node version: 14.9.0

**Comment by FibreFoX** [comment]
The same is with other versions:
https://unpkg.com/jimp@0.13.0/browser/lib/jimp.min.js (shows up as v0.12.1 in the header-comment/preamble)
https://unpkg.com/jimp@0.14.0/browser/lib/jimp.min.js (shows up as v0.13.0 in the header-comment/preamble)
https://unpkg.com/jimp@0.15.0/browser/lib/jimp.min.js (shows up as v0.14.0 in the header-comment/preamble)
https://unpkg.com/jimp@0.16.0/browser/lib/jimp.min.js (shows up as v0.15.0 in the header-comment/preamble)

Seems to be broken for a longer time.

**Comment by qwazwsx** [comment]
Bump

**Comment by ffflorian** [comment]
Ping @oliver-moran

**Comment by disarticulate** [comment]
@qwazwsx appears to be an issue with es6 codes, where uglify is stuck forever on es5.

Easiest way to route around was:
``` browser-buid.js
// const UglifyJS = require('uglify-js');
// npm install terser -D 
const { minify } = require("terser");
...
function minifyProcess(code) {
  console.error('Compressing...');
  return minify(code) // UglifyJS.minify(code, { warnings: true }).code;
}
...
        minifyProcess(code).then(({ code }) => {
          fs.writeFile(
            fromRoot('browser/lib/jimp.min.js'),
            licence + '\n' + code,
            err => {
              if (err) throw err;
              console.error('browserifyed jimp.min.js done.');
            }
          );
        })
```
https://gist.github.com/disarticulate/8ca96e37b45fdbca367a2e7079e5e211

**Comment by mkmorfess** [comment]
is this being addressed at all?

**Comment by hipstersmoothie** [comment]
Jimp no longer ship a .min. jimp.js itself is minified now. our module build should be fixed now too #1149
